bootstrap/005_table_migrations.sql

```diff
@@ -50,16 +52,17 @@ BEGIN
   let alter_table_add_column string := '';
   let alter_table_drop_column string := '';
 
-  if (columns_to_add <> '') then
-    alter_table_add_column := ' ADD ' || columns_to_add;
-    execute immediate alter_statement || alter_table_add_column;
-  end if;
-
+  -- Drop columns first, in case we are re-creating the same columns but in a different order.
```

Comment:
I am not sure how this works...I also don't think we can drop columns on these tables...thats effectively deleting data

Reply:
Crud, you're right. I was thinking that we would just re-materialize, but that won't be happening automatically for the unlucky folks who have some materialized data since v1.19.